packages/ui-products/src/components/ProductForm.tsx

```
@@ -482,7 +503,7 @@ const Form = (props: Props) => {
               <td>
                 <FormControl
                   name="name"
-                  value={(variants[item] || {}).name || ""}
+                  value={(variants[item] || {}).name || ''}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)